cli/src/helpers/data_loader.py

```diff
@@ -6,12 +6,12 @@
 import jinja2
 from jinja2 import Template
 
-from cli.helpers.yaml_helpers import safe_load_all, safe_load
-from cli.helpers.objdict_helpers import dict_to_objdict
+from cli.src.helpers.yaml_helpers import safe_load_all, safe_load
+from cli.src.helpers.objdict_helpers import dict_to_objdict
 
 
 BASE_DIR_PROD = '/epicli'
-BASE_DIR_DEV = os.path.join(os.path.dirname(__file__), '../../')
+BASE_DIR_DEV = os.path.join(os.path.dirname(__file__), '../../../')
```

Comment:
I don't think using relative paths is a good idea. How about defining `BASE_DIR_DEV` in one of the files inside `cli/src` dir and import it here? You would not need to add '../' then.
For example put it in config.py and move config.py to `cli/src`. I think that's a good idea for these reasons:
- Config is not a helper, it's an essential class
- Config does not need separate directory
- No matter how you change the file layout `src/` will always be one directory below the base dir.

Reply:
Agreed about moving Config.py. 

When it comes to BASE_DIR_DEV we could also just make it fixed as BASE_DIR_PROD as these are just the locations in which the epiphany content lives. For production container its just in /epicli:

![image](https://user-images.githubusercontent.com/11056665/153183438-ab7a19ae-d432-4882-aba5-1c4c9869c9b5.png)

For our development container it used to be the case that VSCode would generate some random hash folder if I remember correctly. But that has changed already a long time ago so we could just use: /workspaces/epiphany

![image](https://user-images.githubusercontent.com/11056665/153183808-ab280c4f-e9bb-44dc-a70e-b70c8ad6ae21.png)

For me no need to move these constants for now.

Reply:
Regarding Config.py, we can probably move Log.py as well to /cli/src